program_transformers/src/token_metadata/v1_asset.rs

```diff
@@ -411,3 +427,34 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
 
     Ok(Some(DownloadMetadataInfo::new(mint_pubkey_vec, uri)))
 }
+
+async fn upsert_asset_v1_account_attachments<T: ConnectionTrait + TransactionTrait>(
+    conn: &T,
+    mint_pubkey: &Pubkey,
+    slot: u64,
+) -> ProgramTransformerResult<()> {
+    let edition_pubkey = MasterEdition::find_pda(mint_pubkey).0;
+    let mint_pubkey_vec = mint_pubkey.to_bytes().to_vec();
+    let attachment = asset_v1_account_attachments::ActiveModel {
+        id: ActiveValue::Set(edition_pubkey.to_bytes().to_vec()),
+        asset_id: ActiveValue::Set(Some(mint_pubkey_vec.clone())),
+        slot_updated: ActiveValue::Set(slot as i64),
+        // by default, the attachment type is MasterEditionV1
+        attachment_type: ActiveValue::Set(V1AccountAttachments::MasterEditionV1),
+        ..Default::default()
+    };
+    let query = asset_v1_account_attachments::Entity::insert(attachment)
+        .on_conflict(
+            OnConflict::columns([asset_v1_account_attachments::Column::Id])
+                .update_columns([
+                    asset_v1_account_attachments::Column::SlotUpdated,
+                    asset_v1_account_attachments::Column::AssetId,
+                ])
+                .to_owned(),
+        )
+        .build(DbBackend::Postgres);
```

Comment:
Since the mint is never updateable we should only only be adding the asset_id. Leave the slot updated to the edition.